### ISSUE TITLE
`zola serve`: option to store html files on disk.

### ISSUE THREAD

**Issue by ppom0** [Keats/gutenberg#2377]
I'm using TailwindCSS as a CSS framework. It generates CSS classes definitions from the CSS classes present in the HTML.

If I configure TailwindCSS to look after the `templates` folder, it works fine. I use `tailwindcss --watch`, which generated its CSS to the `static` folder, and `zola serve`, which put assets in the `public` folder.

But when the CSS class names are dynamically generated by Zola macros, they're not present in the `templates` folder: only in the `public` folder, where generated HTML files live.

Problem is: only `zola build` actually write those files, `zola serve` store them in-memory, thus making them inaccessible to `tailwindcss`. It would be super useful to store to have an option like `--html-on-disk` to the `serve` subcommand for those kinds of workflows.

**Comment by ppom0** [comment]
https://zola.discourse.group/t/serve-option-to-store-html-files-on-disk/1975

**Comment by ghost** [comment]
That would be a really awesome feature to have. Would you mind creating a PR with the current Zola version?
Maybe it’ll be easier for Keats to review it…

**Comment by Keats** [comment]
Yes you can do a PR for review. I think it's fine to add although keep in mind it's going to be slower than the current approach

**Comment by ppom0** [comment]
Great! I'll update the patch and open the PR in the following days
